avocado/utils/build.py

```diff
@@ -10,17 +10,18 @@
 # See LICENSE for more details.
 #
 # Copyright: Red Hat Inc. 2013-2014
+#          : IBM 2016
 # Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
+#       : Praveen K Pandey <praveen@linux.vnet.ibm.com>
```

Comment:
@ldoktor sure i am also don't think this is big change can i remove this and rasie new PR ?